### PR TITLE
chore(markdown): add gitpod buttons open cloud env

### PR DESCRIPTION
**Pull request by Sboonny**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

as mentioned by @shootermv in discord, we don't have gitpod easy to run cloud environment like the main repo. 

here are links for testing

https://github.com/Sboonny/chapter/tree/chore/add-gitpod-cloud-environment
https://github.com/Sboonny/chapter/blob/chore/add-gitpod-cloud-environment/CONTRIBUTING.md

### CODE REVIEW

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -37,6 +37,8 @@ We strictly enforce our ["Code of Conduct"](https://www.freecodecamp.org/code-of
 
 # Contributing Code
 
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/from-referrer/)
```
[the remark this answers is not in the document]
Comment (Sboonny) [reply CONTRIBUTING.md:40]: true, but this won't open the fork. For example, if I fork it, and click the image, I will open the main repo in gitpod.

**Review by ojeytonwilliams — APPROVED** [review]

**Comment by ojeytonwilliams** [comment]
@Sboonny if you find a trick to get this working, awesome, but if not please go ahead and merge this.

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Sboonny) [reply CONTRIBUTING.md:40]: That was quick, https://github.com/gitpod-io/gitpod/issues/6410

I have reached a similar conclusion, the workaround is to not use `/form-referrer/`

**Comment by allella** [comment]
Could we move the button under the "Using GitPod" heading? It seems a bit confusing to put it up there above the bullet points explaining the two options (GitPod vs Traditional)

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Sboonny) [reply CONTRIBUTING.md:40]: I am moving forward with your suggestion, although it isn't ideal. 

most people likely just want to check chapter, so giving them a sandbox is more beneficial, than a link that may not work

**Pull request merged**